### DISCUSSION TITLE
How to generate Executable and Library projects in one repo?

### DISCUSSION THREAD

**Discussion by Vyacguru** [friendlyanon/cmake-init#119] [Q&A]
Hello, i want to move my project to cmake-init structure.
But my structure is:

```
repo/
---.github/
---CmakeLists.txt
---MyProjectExe/
------CmakeLists.txt
---MyProjectLib/
------CmakeLists.txt
---MyProjectTool/
------CmakeLists.txt
```
How to make same repo with cmake-init?

**Comment by friendlyanon** [comment]
You might want to take a look at the [Examples](https://github.com/friendlyanon/cmake-init/wiki/Examples) page or any of my recent, CMake managed C++ and C repositories. One of the examples has a build tool as well, so everything should be covered there.
